arangod/Pregel/Algos/WCC.cpp

```diff
@@ -124,9 +140,9 @@ struct WCCComputation
 
       sendMessage(edge, message);
     }
-    // Also send to INBOUND neighbors
-    for (auto const& target : myData.inboundNeighbors) {
-      sendMessage(target, message);
+
+    for (auto pid : myData.inNeighbors) {
```

Comment:
The value_type of `inNeighbors` is `PregelID`, which is a struct consisting of an std::string and a uint16_t. We should not copy the struct values while iterating over the container.
```suggestion
    for (auto const& pid : myData.inNeighbors) {
```